src/core/evaluator.js

```diff
@@ -19,7 +19,7 @@
            MurmurHash3_64, Name, Parser, Pattern, PDFImage, PDFJS, serifFonts,
            stdFontMap, symbolsFonts, getTilingPatternIR, warn, Util, Promise,
            RefSetCache, isRef, TextRenderingMode, IdentityToUnicodeMap,
-           OPS, UNSUPPORTED_FEATURES, UnsupportedManager, NormalizedUnicodes,
+           OPS, UNSUPPORTED_FEATURES, NormalizedUnicodes,
```

Comment:
Can we move `IDENTITY_MATRIX` to this line so `getFontType` can go on the next line, thereby saving the last line of globals.